### PR TITLE
Add info on automating storage deals

### PR DESCRIPTION
**Pull request by ElPaisano**
Addresses https://github.com/filecoin-project/filecoin-docs/issues/1759

1759 has been rescoped. Description (copied from issue) is below:

_Rescoped after discussion. Tutorials are not yet available. [Deal making tutorial - client contract#1761](https://github.com/filecoin-project/filecoin-docs/issues/1761) scope and cover the client contract tutorial work_

_Scope:_
_In the Fundamentals section, explain the available smart contract methods to automate storage deals._

_@ElPaisano suggestion: this might be better in Developing contracts since it discusses two contract methods_

### CODE REVIEW

**Comment by ElPaisano** [comment]
@trruckerfling can you review this when you have a moment? Thank you

File: content/en/smart-contracts/fundamentals/fvm-storage-deals/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+type: docs
+menu:
+  smart-contracts:
+    parent: "smart-contracts-developing-contracts"
+    identifier: "storage-deals-fvm"
+weight: 500
+toc: true
+---
+
+Using the programmatic storage building blocks provided by the FVM, developers can build:
```
[the remark this answers is not in the document]
Comment (ElPaisano) [reply content/en/smart-contracts/fundamentals/fvm-storage-deals/index.md:17]: Suggestion makes sense @trruckerfling, this is my proposed adjustment based on that 

```suggestion
Using the programmatic storage building blocks provided by the FVM, developers can build data-centric solutions. Use cases include, but are not limited to:

```

**Comment by johnnymatthews** [comment]
I feel like this PR is out-of-date by now. @trruckerfling @ElPaisano can you confirm? If it's stale I'll delete this PR and branch.

**Comment by ElPaisano** [comment]
I'm unsure, but there's been no action on it on my end. @trruckerfling?

**Pull request closed without merging**